apps/storefront/components/Image/Image.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cn from 'classnames';
+
+import classes from './Image.module.css';
+
+interface ImageProps {
+  alt: string;
+  src: string;
+  boxShadow: boolean;
+}
+
+const Image = ({ alt, src, boxShadow, ...rest }: ImageProps) => {
+  return (
+    <div className={cn(classes.container, { [classes.boxShadow]: boxShadow })}>
+      <img
```

Comment:
Should we not use NextJS [Image](https://nextjs.org/docs/pages/api-reference/components/image) component for optimisation?

Reply:
Fair enough!